src/Browser.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Panther project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Panther;
+
+final class Browser
```

Comment:
`Browers` for consistency with sulfony's `Events`?

Reply:
I agree, I will fix this if we indeed want this PR to go to master.